hw/ip/aes/data/aes.hjson

```diff
@@ -429,46 +429,30 @@
     fields: [
       { bits: "0",
         name: "START",
+        resval: "0"
         desc:  '''
           Keep AES unit paused (0) or trigger the encryption/decryption of one data block (1).
           This trigger is ignored if MANUAL_OPERATION=0 (see Control Register).
         '''
         tags: ["excl:CsrNonInitTests:CsrExclWriteCheck"]
       }
       { bits: "1",
-        name: "KEY_CLEAR",
+        name: "KEY_IV_DATA_IN_CLEAR",
```

Comment:
Im ok with this name, but it could be something simpler like
Input_regs_clear

Reply:
I was thinking about that, too. But my fear is that people might erroneously think `INPUT_REGS_CLEAR` only deals with input data. Thus I thought it would be better to be as explicit as possible.

Reply:
that it fine with me